tests/NuGet.Services.AzureSearch.FunctionalTests/Support/RelevancyFactAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+
+namespace NuGet.Services.AzureSearch.FunctionalTests
+{
+    public class RelevancyFactAttribute : FactAttribute
```

Comment:
Keep in mind that if you'd want to add another variant of it descending from `TheoryAttribute`, you'd need to specify test discoverer explicitly (like we do in [NuGet.Jobs](https://github.com/NuGet/NuGet.Jobs/blob/a48031f29aec0e3afd5efc7db425649869567029/tests/TestUtil/AdminOnlyTheoryAttribute.cs#L15)), otherwise it would fail to run the tests properly.
Direct descendants of `FactAttribute` does not seem to suffer from it, but I would still specify it here, too, for consistency.